src/vos/vos_ts.h

```diff
@@ -434,6 +434,9 @@ vos_ts_set_add(struct vos_ts_set *ts_set, uint32_t *idx, const void *rec,
 	if (idx == NULL)
 		goto calc_hash;
 
+	if (ts_set->ts_init_count == ts_set->ts_set_size)
+		return -DER_BUSY; /** No more room in the set */
+
```

Comment:
Just curious, when does this happen? How will we cope with it?

Reply:
My thought process was that when I add support for iteration, I may allocate some fixed number and then periodically flush the set by updating the read timestamps.   I haven't implemented that yet so it's subject to adjustment.

Reply:
I see. Thanks for elaborating.